lte/gateway/c/core/oai/lib/openflow/controller/PagingApplication.cpp

```diff
@@ -44,12 +44,16 @@ void PagingApplication::event_callback(const ControllerEvent& ev,
   struct in6_addr dest_ipv6;
 
   if (ev.get_type() == EVENT_PACKET_IN) {
-    OAILOG_DEBUG(LOG_GTPV1U, "Handling packet-in message in paging app\n");
     const PacketInEvent& pi = static_cast<const PacketInEvent&>(ev);
     of13::PacketIn ofpi;
     ofpi.unpack(const_cast<uint8_t*>(pi.get_data()));
-    handle_paging_message(ev.get_connection(),
-                          static_cast<uint8_t*>(ofpi.data()), messenger);
+    OAILOG_DEBUG(LOG_GTPV1U,
+                 "Handling packet-in message in paging app: tbl: %d\n",
+                 ofpi.table_id());
+    if (ofpi.table_id() == 0) {
```

Comment:
Can we define a macro here instead of `0`?

Reply:
We have not done it anywhere in GTP application, have a look at call create_default_flow_mod(). '0' is passed from all GTP apps. I can post other incremental PR for this improvement.

Reply:
Sure, created https://github.com/magma/magma/issues/11643 and marked as a `good-first-issue` :)